SAMtemplates/alarms/main.yaml

```diff
@@ -81,3 +85,24 @@ Resources:
         - !ImportValue lambda-resources:SlackAlertsSnsTopicArn
       OKActions:
         - !ImportValue lambda-resources:SlackAlertsSnsTopicArn
+
+  PSUConcurrencyAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties: 
+      AlarmName: !Sub "${StackName}-Concurrency-Alarm"
+      AlarmDescription: "Monitors the concurrency of the PSU as a whole and triggers when concurrency exceeds the threshold."
+      Namespace: "AWS/Lambda"
+      MetricName: "ConcurrentExecutions"
+      Statistic: Sum
```

Comment:
I havent checked the docs for the concurrency metric but i suspect we dont want to sum them in this case as i would assume the metric is reporting the current number of executions in an interval so would maybe want maximum instead, rather than it collating counts of events like errors.

Reply:
You've hit on something I spent a while arguing with myself here - I think they're two different things that we could use to answer the underlying question. I think sum answers "How many total users have there been in the last minute?" and maximum would be "what's the peak number of users over the last minute?", and the question we actually want to answer is "Are we seeing a suspicious amount of traffic?". 

That said, more than happy to go with the maximum. At the very least, I'll try a run on it and see what the numbers look like. Might be worth an elab, tbh.

Reply:
So i think as were comparing the value to a limit, we want to know what the actual current number is and how close it is to the limit. But its not a bad shout to see what others think in elab.

Reply:
Since it's such a minor change, I suggest we merge this as maximum, and raise it in elab. If we decide sum is better, I'll make a new PR?